### ISSUE TITLE
active_callback operator using '<=' or '>='

### ISSUE THREAD

**Issue by designaction-ria** [themeum/kirki#1427]
### Issue description:
The latest Kirki update seems to have switched the operator logic for greater than/less than or equal to.

I have a number slider (from 1 through 7) that controls the visibility of other fields. For example, if the number slider is set to 3, `field_1`, `field_2`, and `field_3` would show, but `field_4` through `field_7` would be hidden. (Field names are just to illustrate/simplify).

Up until the latest update, this has worked perfectly using the `<=` operator on each field, and checking the number slider value within the `active_callback`. However, with the latest release, I had to switch to the `>=` operator to get it to work—in essence, the logic has switched.

There also appears to be different logic used during the page refresh—if I have the operator set to `<=` and I change the number slider, I'll see the correct logic during the page refresh, but once the refresh is complete it will switch to using the incorrect logic.

### Version used:
3.0.4

### Using theme_mods or options?
`theme_mod`

### Code to reproduce the issue (config + field(s))
Example number slider (setting number of slides to show):
```
Kirki::add_field( 'NAMESPACE', array(
	'type'        => 'slider',
	'settings'    => 'NAMESPACE_slides_slide_number',
	'label'       => esc_attr__( 'Number of slides', 'NAMESPACE' ),
	'section'     => 'NAMESPACE_slides',
	'default'     => 1,
	'choices'     => array(
		'min'  => 1,
		'max'  => 7,
		'step' => 1,
	),
) );
```

Example field, using the `<=` operator that is no longer working correctly in the latest update:
```
Kirki::add_field( 'NAMESPACE', array(
	'settings' => 'NAMESPACE_slides_slide_2',
	'label'    => __( 'Slide 2', 'NAMESPACE' ),
	'section'  => 'NAMESPACE_slides',
	'type'     => 'editor',
	'priority' => 30,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => 'NAMESPACE_slides_slide_number',
			'operator' => '<=',
			'value'    => 2,
		),
	),
) );
```

**Comment by aristath** [comment]
I just tested your code and it looks to me like it's working perfectly fine.
The full code I used:
```php
Kirki::add_config( '_s' );
Kirki::add_section( 'theme_options', array(
	'title' => esc_attr__( 'Theme Options', '_s' ),
	'icon'  => 'dashicons-admin-customizer',
));
Kirki::add_field( '_s', array(
	'type'        => 'slider',
	'settings'    => '_s_slides_slide_number',
	'label'       => esc_attr__( 'Number of slides', '_s' ),
	'section'     => 'theme_options',
	'default'     => 1,
	'choices'     => array(
		'min'  => 1,
		'max'  => 7,
		'step' => 1,
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_2',
	'label'    => __( 'Slide 2', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 30,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '<=',
			'value'    => 2,
		),
	),
) );
```
When `_s_slides_slide_number` is `1` or `2`, `_s_slides_slide_2` is shown.
When `_s_slides_slide_number` is anything above 2, `_s_slides_slide_2` is hidden.
A rough translation of the condition you've got there in `active_callback` would be this:
```php
if ( get_theme_mod( '_s_slides_slide_number' ) <= 2 ) {
      // SHOW CONTROL
} else {
    // HIDE CONTROL
}
```

**Comment by designaction-ria** [comment]
Thanks for looking into this! Unfortunately I'm still seeing the issue on my end—let me see if I can do a better job explaining what's going on.

The issue seems to kick in once there are three or more sections (not just two). See this GIF (and watch the slide numbers change):
![active_callback](https://user-images.githubusercontent.com/22329203/27607271-3ac1ac6c-5b38-11e7-912e-69a7d4623acf.gif)

You'll see there are a number of slide fields—changing the number slider changes what number of Slide editors are available for updating.

You can see that when I drag the slider, the conditional fields immediately shift—once the screen finishes refreshing, the available fields again change. The during-refresh and after-refresh states seem to follow the opposite logic—when I use `>=`, fields show correctly after the page refresh but incorrectly while loading, and the opposite occurs when using `<=`. Prior to this recent update, both were in sync—ie., I'd move the slider, the correct fields would show while waiting for the page refresh, then the correct fields would still show (no change). After the recent update, they now behave with opposite logic.

**Comment by aristath** [comment]
Can you please post the code for ALL those fields so that I may replicate this? In your previous reply you only included 1.

**Comment by designaction-ria** [comment]
Sure, here's a simplified version based on your example:

```
Kirki::add_config( '_s' );
Kirki::add_section( 'theme_options', array(
	'title' => esc_attr__( 'Theme Options', '_s' ),
	'icon'  => 'dashicons-admin-customizer',
));
Kirki::add_field( '_s', array(
	'type'        => 'slider',
	'settings'    => '_s_slides_slide_number',
	'label'       => esc_attr__( 'Number of slides', '_s' ),
	'section'     => 'theme_options',
	'default'     => 1,
	'choices'     => array(
		'min'  => 1,
		'max'  => 5,
		'step' => 1,
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_1',
	'label'    => __( 'Slide 1', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 10,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '>=',
			'value'    => 1,
		),
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_2',
	'label'    => __( 'Slide 2', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 20,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '>=',
			'value'    => 2,
		),
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_3',
	'label'    => __( 'Slide 3', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 30,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '>=',
			'value'    => 3,
		),
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_4',
	'label'    => __( 'Slide 4', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 40,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '>=',
			'value'    => 4,
		),
	),
) );
Kirki::add_field( '_s', array(
	'settings' => '_s_slides_slide_5',
	'label'    => __( 'Slide 5', '_s' ),
	'section'  => 'theme_options',
	'type'     => 'editor',
	'priority' => 50,
	'default'  => '',
	'active_callback'    => array(
		array(
			'setting'  => '_s_slides_slide_number',
			'operator' => '>=',
			'value'    => 5,
		),
	),
) );
```

**Comment by aristath** [comment]
OK, I see the problem now.

**Comment by aristath** [comment]
Fixed, will be included in 3.0.8.

**Comment by designaction-ria** [comment]
Great, thank you so much!
